pom.xml

```diff
@@ -381,6 +412,10 @@
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
+                    <doclint>all,-missing</doclint>
```

Comment:
I just filed some hackathon tickets to actually clean up the javadoc errors.  Not sure we want to turn this on just yet.